server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/SetMailboxesUpdateProcessor.java

```diff
@@ -71,10 +73,14 @@ private void handleUpdate(String mailboxId, MailboxUpdateRequest updateRequest,
         try {
             validateMailboxName(updateRequest, mailboxSession);
             Mailbox mailbox = getMailbox(mailboxId, mailboxSession);
+            checkRole(mailbox.getRole());
             validateParent(mailbox, updateRequest, mailboxSession);
 
             updateMailbox(mailbox, updateRequest, mailboxSession);
             responseBuilder.updated(mailboxId);
+
+        } catch (SystemMailboxException e) {
+            notUpdated(mailboxId, "invalidArguments", "Cannot rename a system mailbox.", responseBuilder);
```

Comment:
It will also generate the same exception so the description message has been changed in : "Cannot update a system mailbox."